google/cloud/internal/throw_delegate.h

```diff
@@ -27,8 +27,9 @@ namespace internal {
  * @name Delete exception raising to hidden functions.
  *
  * The following functions raise the corresponding exception, unless the user
- * has disabled exception handling, in which case they print the error message
- * to std::cerr and call std::abort().
+ * has disabled exception handling, in which case they call the function set by
+ * google::cloud::SetTerminateHandler() (by default print the error message
```

Comment:
Consider:

```C++
/*
...
 * `google::cloud::SetTerminateHandler()`. The default handler prints the error
 * message to `std::cerr` and calls `std::abort()`.
*/
```

Note that Doxygen uses markdown formatting so things like code look better with the backticks.

Reply:
Done